rocAL/rocAL_pybind/amd/rocal/decoders.py

```diff
@@ -18,8 +19,9 @@ def image(*inputs, user_feature_key_map = None, path='', file_root ='', annotati
             "loop": False,
             "decode_size_policy": types.MAX_SIZE,
             "max_width": 0,
-            "max_height":0}
-        decoded_image = b.COCO_ImageDecoderShard(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+            "max_height": 0}
+        decoded_image = b.COCO_ImageDecoderShard(
+            Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
```

Comment:
Can't we have this in previous line itself?
`decoded_image = b.COCO_ImageDecoderShard(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))`

Reply:
Changed for all